internal/command/volumes/extend.go

```diff
@@ -72,7 +73,22 @@ func runExtend(ctx context.Context) error {
 		return err
 	}
 
-	sizeGB := flag.GetInt(ctx, "size")
+	sizeFlag := flag.GetString(ctx, "size")
+	sizeBytes, err := units.FromHumanSize(sizeFlag)
+	if err != nil {
+		return fmt.Errorf("invalid size: %w", err)
+	}
+
+	sizeGB := int(sizeBytes) / units.GB
```

Comment:
what happens if `--size=3` (no units) is passed as value?

Reply:
I thought of this algorithm:

If it's not 0 and it has a `+` sign (or not), check if it's just an `int` (after `+` sign).
If just an int, set volume size.
If not just an `int`, try parsing with `FromHumanSize` and then set volume size.

Reply:
Implemented that